basic-steps/src/main/resources/org/jenkinsci/plugins/workflow/steps/DeleteDirStep/help.html

```diff
@@ -0,0 +1,6 @@
+<div>
+    Recursively deletes the current directory and its contents.
+    Symbolic links and junctions will not be followed but will be removed.
+    To delete a specific directory of a workspace wrap the <code>deleteDir</code>
```

Comment:
this was discussed [further up](https://github.com/jenkinsci/workflow-plugin/pull/204/files#r38955418)